x/wasm/client/proposal_handler_test.go

```diff
@@ -57,15 +57,51 @@ func TestGovRestHandlers(t *testing.T) {
wWREdt7SzvHNtc5zz7fMcCd719ka1hDnKsFZrszRUttqAjZG+1PWJ/zP4V4XHPH+zvOz8UzjomvV8dHbv1tv8S3HfZrZ++s37R/6iqvnjH+x+0BDbtvyr+jV8cv+32u+5+7OvPPPviS9/9h396+5eTINTUrg61d0XXbei76Krjt9/92NeffObZl/7hlVff/iUIlVW0Nbqup7fvoquT+Phdn7/vu6+8WlmzuqdvXxLfdtdjTz7z7IvffevtX54WKmt6+pK4OH78b597/sQbb55+79gNtz748PMnXnz5lVd/+rPt937rBy+98mpf/659l18dv/n2O77+1DdPvPDSy2/WzJl75f5/+93ZyWJl7uA/vVVV3aBq9Yvih6//m689+9wf5sxd3NC7rX/XZVfsv/r6I9948fU3/vH0e/9XN+4wC391TyC+siXwyNe+eeLlV99863Ob77k3eEfDxAuvTPbvuuJKt6daXBX49buq1rVhU3fPnXednRxIF77z3ZOv/fgn/3x2ElB86dG3hKNbPQsFV834X1cVH+9YPLt4yjHfYxMCQrvgdtjcLneNb3d1rXvQYxfqfV6Hx+F2EOXwC05HhctWNdsZddS4XU7Rvc9td8+u3C1scTQ7bEKNq9ofFRatiKOccGBF8TvOo084FriOfui43D27Yq63zl/nP+DyuRa4Lnevdfb6mgS/YHOEKpqEBa4KR/GvXT5XQ2yH0OwwPesd1Y717rBnrfPoZM08T6Cm2YHEJdXFTwtH75lfMftTn3UGnOvc9qp53uITQdNf/NECv7M46Sw+5Vzu//xLvv8eFI/f5ujyju+vKz7tKb4c7BB8rrCn1+N3mRWLHVcIl3uLx+bV++Z4t3uKt7gef9A/Vwh9SRj/6Uq33+ksPlQznhNHG9e4dgjF24Ti846FjupKcNlsDptg93i8dp+zwi7aaoRa56yaOvts+1z7fKHeuciz3HbAMWx/zv6a/XX/G94f2d+0/9R2yvlz+zv2X6PTwu9df7D7V63b2L/rji9+8X6X29u5YeOlvzn5mlA3r7Pr0n3f6njl1VO1N918+xenxI9IX/+uJL5i/1PfXFj/6GM//om36867HnX71m1MKXfcrcV//e4Vic99flXj4H0PfOnLDz7y6Fefee7brgr/7EXRTT2XPPzI93/wgHv+gqUrNm568SUBLVuxsrEtHN1+0Y7dA4OXEskaknFq2Bg9fOSWB7/ytScmTvr1Lz/4ve+r2tVLr3U6hGYHFmyBluLRRY5Qdb2w3LvYuda5VahaU/yKa7mwXGj0tFfYineOd3nn+DzFeyMO2eMNznEucSx02jaHhYudAcHn9ro3o1WC39vpiDoXuAW/e3dfV1tlm7vF4xtf2VAcb/SsKd4419svLK7eUjXf7XNt867yFio2xda41jl9rktcNqfocBZvHULbPb7iw1cv7anwuSpnRdy+ziZBLE6sTw74bcUHehZu9wwsioy7e32LHNv6uhxVHp8r4vaNd84vvtxaeezzqUJF8du3PHc8cMdrR7d96e+ORtxrBPvQSl+vr9E56+gTV+KLhYi7ZrNQf88Zz9HfPFQ53hBy1Aie8U/fLAw7Kx1et/iZoeK/+QyPOre3+Lk6/z7v/OJN49scN3RXz75h99riG82OBYJ9PB912m6A4purdwg+wX6sZuuODcX/tt5lEwadC9vt41VNQtJ/qa/ScZVjQ2RRZZPgddurXMXPHfuRr9JR4TT9l7v9dX4h7PG5Gj2L94w4Qo7iw/46weF0VLpuOHAuI8udh8u7WHwa5n8fZMERHpuV+iLZkvdLqT855WXzt3/ycVOjjnU8i9WZfXOkrJ69cL6jdm4NQNUciN18egIq7K5ZEPsiefLZ3T7w+BfVz1sAsWdom9MGsVfJk586oBXghTqohNgZ2lpdC7Gb3puAClsDxO4hD0Jfb2oRxB4hz/ZbNkLsKfp08QDEvk2eHOll3RD7IXl0b6Au7DKI/U/aswDQDLHfUKyj/eshdvRfJ6BilkwvpK/gDu94J4ADYveTplrrbU9gzvEs8EDsedLi3AI63AyxHxCgxk0CX+e8TdB1k3dzEmAJxP6FNAg+gBjE3ifPW/wQpFHJ/JGDMLStY2sLgBmpwU2XXH2EBMXtIR798gtFCC+AT8XXDitX7brqeHFNpm4dzIXY934zARXLBllMc8xa3eWsWKkunEMfchfVQ+wMQb14OcB1sAeuddHr9u3k5/rGQ6uyq2HplVVS5aX5hHADGrUDXLN4zLaXXm3eaFwGO3u0igMArdkdADffAnDLLYTUP/7tBFSc9Nj4tNbfu9TzcjMvn+LlCV6ecbDytJuVk+4ZVGWJB+ACCvx1NAAoHuFwkRUTk2zm4iQfsGgNxBBtiCPCl1nxwRGOSN2TkpmL1mV1FRv+elomEXupffPZasRjn2Dp1vZPb3GI/h7fb004wctTFkvYSk/7Sbk1OETjnhc2sHIxKvJygpZnl5ylJZS8VU8m2OqxJgI4bU1qBzgyOfn7n7GdFM+w8tRz1bz/EqB51vXrAP65miffYeL++6kuTLz/7ka+4E0lAwO7uBYBzpydBJic+pIPYSD5tZXwgNRbUmDxgpWbv2yR3MOjPT+1DXCGL5BGBPWW1Py9bYr5DWCDIkw0NNC7gOK7DWz+BgQ3RuxUjckvBDkLrifR4weTkx/kZsgANHKpO/0qX9wDdoDfTX64RHSwlY0yWShOcpmY4DLxr7S8ySZU8QbRw+RT5Ah1bAen53L8hrdOTZ4lK5zBujPViOF/p41VP8EoXfzdJoFVbOb8Zxsp/nCzndXv5pS7jHH0w9RZOtCZMVY/wTd4lpeTU9rBH4pByhlHUWU8vYmPeyfA5iO/bDhzPwN//xArzzzCeq1/ghP0SU7HEwBkW5MnajiBWHHfC3y8k+Tngb93n2HycYSXk+8/v4Ry/AO+rkmOP8ky0ESGzky+zyQKSu/HIPbA+AtQYbP9vwAAAP//BBDwpLpnAgA=",
```

Comment:
:exploding_head:  can you just load a file here or something?

This is fine, but a bit hard to update

Reply:
👍 this is just a random byte array in this test. I will make it clear.

Reply:
Merging this now as another PR depends on the changes. If the solution here does not fit your expectations, please let me know that I fix it with a new PR.